Classes/PrepareContentRelease/Dto/ContentReleaseMetadata.php

```diff
@@ -72,20 +84,22 @@ public static function fromJsonString($metadataEncoded, ContentReleaseIdentifier
             NodeRenderingCompletionStatus::fromString($tmp['status']),
             isset($tmp['manualTransferJobIds']) ? array_map(function (string $item) {
                 return PrunnerJobId::fromString($item);
-            }, json_decode($tmp['manualTransferJobIds'])) : []
+            }, json_decode($tmp['manualTransferJobIds'])) : [],
+            $tmp['workspace'] ?? 'live'
```

Comment:
I'd suggest we remove the Workspace from the full ContentReleaseMeta; and I'd rather use the additional values here (in order to simplify APIs)